lib/rules/media-feature-name-case/__tests__/index.js

```diff
@@ -163,6 +163,27 @@ testRule({
 			line: 1,
 			column: 16,
 		},
+		{
+			code: '@media not all and (\\MONOCHROME) { }',
+			fixed: '@media not all and (monochrome) { }',
+			message: messages.expected('MONOCHROME', 'monochrome'),
+			line: 1,
+			column: 21,
```

Comment:
[suggest] Could you test also `endLine`/`endColumn` since passing `index`/`endIndex` to `report()`? It's OK only for this PR's addition.

```suggestion
			column: 21,
			endLine: 1,
			endColumn: 32,
```